tests/formattedcode/test_output_templated.py

```diff
@@ -114,17 +114,29 @@ def test_custom_format_with_custom_filename_fails_for_directory():
 def normalize_quotes(s):
     return s.replace("'", '"')
 
+@pytest.mark.scanslow
+def test_scan_custom_html_output_for_a_directory():
+    test_dir = test_env.get_test_loc('templated/tree/scan/')
+    custom_template = test_env.get_test_loc('templated/sample-template.html')
+    expected_file = test_env.get_test_loc('templated/tree/expected.html')
+    result_file = test_env.get_temp_file('html')
+    args = ['-clip', '--strip-root', '--custom-template', custom_template, '--custom-output', result_file, test_dir]
+    run_scan_click(args)
+    results = open(result_file).read()
+    expected = open(expected_file).read()
+    assert expected == results
 
 @pytest.mark.scanslow
 def test_custom_format_with_custom_filename():
     test_dir = test_env.get_test_loc('templated/simple')
     custom_template = test_env.get_test_loc('templated/sample-template.html')
+    expected_file = test_env.get_test_loc('templated/simple-expected.html')
     result_file = test_env.get_temp_file('html')
-    args = ['--info', '--custom-template', custom_template, '--custom-output', result_file, test_dir]
+    args = ['-clip', '--custom-template', custom_template, '--custom-output', result_file, test_dir]
     run_scan_click(args)
     results = open(result_file).read()
-    assert 'Custom Template' in results
-    assert __version__ in results
+    expected = open(expected_file).read()
+    assert expected == results
```

Comment:
That's a tiny nit for the future, but we now use the form   `assert results == expected` since Pytest changed the way it displays assertions failure traces.